common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -97,10 +97,14 @@ private void tryUserToggle() {
     }
 
     @Override
-    public int compareTo(Feature other) {
+    public int compareTo(Object other) {
+        if (!(other instanceof Feature feature)) {
+            return 0;
```

Comment:
I thought 0 means "is equal to"? This looks wrong. Should you not sort stuff so everything that is not a feature will come before, or after, all features?

Reply:
I fixed the logic itself, this change was dumb to begin with.